media-libs/libsoundtouch/libsoundtouch-2.1.0.recipe

```diff
@@ -0,0 +1,74 @@
+SUMMARY="A library to adjust the sound tempo, pitch and playback rate"
+DESCRIPTION="SoundTouch is an open-source audio processing library for \
+changing the tempo, pitch and playback rates of audio streams or audio files. \
+The library additionally supports estimating stable beats-per-minute rates for \
+audio tracks.
+
+- Tempo (time stretch): Changes the sound to play at faster or slower tempo \
+than originally without affecting the sound pitch.
+- Pitch (key) : Changes the sound pitch or key while keeping the original \
+tempo (speed).
+Playback Rate : Changes both tempo and pitch together as if a vinyl disc was \
+played at different RPM rate."
+HOMEPAGE="https://www.surina.net/soundtouch"
+COPYRIGHT="2001-2018 Olli Parviainen"
+LICENSE="GNU LGPL v2.1"
+REVISION="1"
+SOURCE_URI="https://gitlab.com/soundtouch/soundtouch/-/archive/$portVersion/soundtouch-$portVersion.tar.bz2"
+CHECKSUM_SHA256="3fb740c7ac4b304342fbf11a91d14fa3e296269b9fec6266609c4620477d0ad6"
+SOURCE_DIR="soundtouch-$portVersion"
+
+ARCHITECTURES="!x86_gcc2 x86 x86_64"
+SECONDARY_ARCHITECTURES="x86"
+
+PROVIDES="
+	libsoundtouch$secondaryArchSuffix = $portVersion compat >= 1
+	cmd:soundstretch$secondaryArchSuffix
+	lib:libSoundTouch$secondaryArchSuffix = 1.0.0 compat >= 1
```

Comment:
wasn't sure about this, as the soname didn't change between versions?

Reply:
PS, why the use of portVersionCompat (I even think the compat >= 1 shouldn't be needed)?
Or would it be required for other recipes depending on compat >=1?

Reply:
Yep, I can confirm that the soname did not change. (It's `libSoundTouch.so.1` for both the old release, `2.0.0`, and the new release, `2.1.0`)
```
$ objdump -x libSoundTouch.so.1.0.0 | grep SONAME
  SONAME               libSoundTouch.so.1
```

Reply:
Shouldn't $portVersionCompat reflect the version of the cmd also then? (like 2.1 instead of 1)

Reply:
I don't really know. Well, I'd suggest setting  `portVersionCompat="$portVersin compat >= 2.1"`, like you.
That said, we could use `$portVersion` (instead of `$portVersionCompat`) and drop the definition of `portVersionCompat`.
Any opinions here?

Reply:
I'm in favour of "portVersionCompat=$portVersion compat >= 2.1" then (as the actual version printed by the binary is 2.1)

Reply:
IDK. Maybe we could simply drop the `portVersionCompat` and not use any compat for the package and for the `cmd:`.
Pros: No need to check for the compatibility of the runtime.